build/azure-pipelines/package-product.yml

```diff
@@ -0,0 +1,30 @@
+# sqlcmd package pipeline
+
+pr: none
+
+variables:
+  Build.SourcesCliDirectory: '$(Build.SourcesDirectory)/'
+
+stages:
+  - stage: CreatePackages
+    displayName: Create and Test Package Matrix
+    jobs:
+    - job: Package
+      strategy:
+        matrix:
+          rpm:
+            imageName: 'ubuntu-latest'
+            os: linux
```

Comment:
I think docker and mac can follow this format, I think windows is the only odd one out.  I was planning to deal with that when I get to the Windows PR, and make the fewest possible changes when I get to the Windows PR (it might just come down to pipeline.sh vs pipeline.cmd etc.!